internal/httpclient/model_recovery_identity_address.go

```diff
@@ -16,8 +16,8 @@ import (
 	"time"
 )
 
-// RecoveryAddress struct for RecoveryAddress
-type RecoveryAddress struct {
+// RecoveryIdentityAddress struct for RecoveryIdentityAddress
+type RecoveryIdentityAddress struct {
```

Comment:
Just to be sure. Are this changes cherry-picked from upstream?

Reply:
Not really. It was change caused by a new `go-swagger` tool. I think this is some kind of optimization since those two structs (`recoveryIdentityAddress` and `recoveryAddress`) are identical.